opal/tools/wrappers/Makefile.am

```diff
@@ -62,7 +62,8 @@ endif # OPAL_INSTALL_BINARIES
 endif # OPAL_WANT_SCRIPT_WRAPPER_COMPILERS
 
 opal_wrapper_SOURCES = opal_wrapper.c
-opal_wrapper_LDADD = $(top_builddir)/opal/lib@OPAL_LIB_PREFIX@open-pal.la
+# XXX(nevans) I need to fix this, compile or runtime for the threading library?
```

Comment:
Per above, please don't leave "to-do"-like comments.  Please actually do whatever it is you need to do and only put the final product up here in github to be merged.  Thanks.